frontend/src/pages/TermPlanner/ImportPlannerMenu/ImportPlannerMenu.tsx

```diff
@@ -76,7 +75,7 @@ const ImportPlannerMenu = () => {
             });
             return;
           }
-          dispatch(updateDegreeLength(fileInJson.numYears));
+          axios.put('/user/updateDegreeLength', { numYears: fileInJson.numYears });
```

Comment:
Should this be async?

Reply:
Also what is the expectation for error handling?

Reply:
async - yes

error handling - throw it in a try-catch -> log on failure - we will come back to make a warning for this eventually

Reply:
Error handling has been added, although I'm not sure how to go about error handling if I move to async?

Reply:
Fixed